### PR TITLE
Router feature

### PR DESCRIPTION
**Pull request by dheerajsachdeva**
##In this milestone 

- add 3 routes Home, Calculator, and quote
- add my own styling to the web page
- create navigation links to the three routes
- use React Router

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,10 +1,30 @@
+// eslint-disable-next-line
+import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
```
Comment (levy002) [inline src/App.js]: - [ ] Kindly make sure to not disable Linters. Linters help us to write clean code.

File: src/components/Navbar.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+// eslint-disable-next-line
```
Comment (levy002) [inline src/components/Navbar.js]: - [ ] Kindly make sure to not disable Linters. Linters help us to write clean code.

File: src/components/styles.css [hunk 3]
```diff
@@ -5,12 +5,66 @@
...
+  margin-top: 30px;
 }
 
 .calculator-grid {
+  width: 50%;
   display: grid;
   margin-top: 3rem;
-  margin-left: 5%;
+
+  /* margin-left: 5%; */
```
Comment (levy002) [inline src/components/styles.css]: - [ ] Kindly make sure to remove all comments in order to keep your codebase clean and professional.

**Review by levy002 — CHANGES_REQUESTED** [review]
## Hello @dheerajsachdeva   , :wave:
### I'm Levy, your PR reviewer

Good job so far :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- Good UI :heavy_check_mark: 

## Status: REQUIRED CHANGES ♻️
_Check the comments under the review._

- [ ] Kindly make sure to remove the file `README1.md` because it doesn't do anything in your repo as long as you have another well-written README file.  Kindly remove it in order to keep your repo clean.

    
    image


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @levy002 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by omarsalem7 — APPROVED** [review]
Hi @dheerajsachdeva 😊, 
# STATUS: APPROVED! 🥇 💯 

giphy

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 🚀 
Congratulations! 🎉 

## Highlights
✅ Project has met all the technical aspects you needed for this project. Keep rocking! 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**